smartsim/_core/_cli/utils.py

```diff
@@ -70,6 +76,7 @@ def pip_install(packages: t.List[str], end_point: t.Optional[str] = None, verbos
 
     if verbose:
         logger.info(f"Installing packages {packages}...")
+    # pylint: disable-next=consider-using-with
     proc = subprocess.Popen(
         cmd_arg, shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE
```

Comment:
We shouldn't have any shell=True commands. Feel free to just open a ticket rather than any more immediate addressing of the comment